index.dd

```diff
@@ -93,29 +93,36 @@ $(DIVC boxes,
                 Adam D. Ruppe's weekly newsletter. $(SPANC twid)
             )
         )
+        $(TOUR graduation-cap, Learn,
+            $(P Take the $(LINK2 http://tour.dlang.org,
+                Tour), explore
+                $(LINK2 $(ROOT_DIR)overview.html, major features) in D,
+                compare D with $(LINK2 $(ROOT_DIR)ctod.html, C) or
+                $(LINK2 $(ROOT_DIR)cpptod.html, C++),
```

Comment:
Best not. Language comparisons are often seen as controversial and attract flame wars.

These pages are meant to provide an intro to D using concepts from other languages, not as a language comparison.